wiki/Ranking_Criteria/en.md

```diff
@@ -36,87 +81,113 @@ All rules are exactly that: **rules**. They are **not** guidelines and may **not
--   **Your map must be perfectly timed.** This means that your BPM and offset are spot-on, sliders end when they should, notes are generally following a recognizable rhythm (such as the lyrics or drums) which is comprehensible by a player, and that there are no unsnapped notes (you can check this by running AIMod (shortcut `Ctrl`+`Shift`+`A`) in the editor).
--   **Uninherited timing sections must be the same in every difficulty** of the mapset. That is, each section must have the same BPM and offset in each difficulty. Furthermore, there must not be extra or missing uninherited sections in any difficulty. A song's timing doesn't change between difficulties, so there's no sense in having different uninherited sections per difficulty.
--   **Uninherited timing sections cannot be used to manipulate slider speeds.** These can be very unexpected for the player while also disrupting the main menu pulse and slightly changing the overall timing of the map.
--   **No two uninherited or two inherited timing sections should be placed at the same point.** An inherited timing section may be placed on an uninherited timing section (but only to change the slider speed). Having two uninherited or two inherited sections on top of each other will cause problems within the beatmap.
+## Audio
 
-### Audio
+### Rules
 
 -   **A beatmap set may only contain one music file used by all difficulties.** Multiple music files within a single beatmap set is unsupported by osu! and results in unexpected behaviour with preview times, metadata, etc.
 -   **The song's audio file must be of reasonable quality.** Try and source `.mp3` files yourself; ripping them from a streaming video site often results in low quality audio with high file sizes. The bitrate of a beatmap's audio file must be no lower than 128kbps and no higher than 192kbps. If you are having trouble acquiring an appropriate audio file, contact one of the more audio-savvy BN; they will be more than happy to help find an `.mp3` for you.
 -   **The minimum draining (play) time for a map is 30 seconds** (but the recommended minimum is 45 seconds). If your map is shorter than that, then try looping the song with an audio editing program. This is so each map gives a high enough score, and also so people do not try and cut songs too short for no reason, thus stopping people from enjoying the song fully.
 -   **You must cut your `.mp3` if you're using less than 80% of it.** This doesn't count intro time, and thus only applies if more than 20% of the outro is unmapped. If you only plan to map a portion of your song, then including the full `.mp3` is a waste of file space. Many `.mp3` editing tools [like this](https://osu.ppy.sh/forum/viewtopic.php?f=56&t=34303) exist which make fading and cutting a song simple. However, if you have a legitimate reason for keeping your full `.mp3` (e.g. a storyboard after the map ends), then that is fine.
```

Comment:
- probably should fix the link to `https://osu.ppy.sh/forum/t/34303` (old site) or `https://osu.ppy.sh/community/forums/topics/34303` (new site)
- change the link text `like this` to `like mp3DirectCut`